src/Microsoft.OpenApi/Validations/Rules/OpenApiInfoRules.cs

```diff
@@ -16,28 +16,26 @@ public static class OpenApiInfoRules
         /// <summary>
         /// Validate the field is required.
         /// </summary>
-        public static ValidationRule<OpenApiInfo> FieldIsRequired =>
+        public static ValidationRule<OpenApiInfo> InfoRequiredFields =>
             new ValidationRule<OpenApiInfo>(
                 (context, item) =>
                 {
 
                     // title
                     context.Enter("title");
-                    if (String.IsNullOrEmpty(item.Title))
+                    if (item.Title == null)
```

Comment:
>m.Title == [](start = 27, length = 10)

so now required fields can be empty, right?

Reply:
Yes, but I have a few more places where it needs to be fixed.